test/joiner_join_snippet_data_test.rb

```diff
@@ -89,7 +93,26 @@ def test_join_all_snippets
         '', '- Did stuff', '')
       add_snippet('20141231', 'mbland', 'Mike Bland', 'michael.bland@gsa.gov',
         'Public', '- Did stuff', '')
-      @impl.join_snippet_data
+      Joiner.join_data @site
+      assert_equal @expected, @site.data['snippets']
+    end
+    # rubocop:enable MethodLength
+
+    # rubocop:disable MethodLength
+    def test_join_all_snippets_when_user_email_is_private
+      self.team = {
+        'mbland' => {
+          'name' => 'mbland', 'full_name' => 'Mike Bland',
+          'private' => { 'email' => 'michael.bland@gsa.gov' }
+        },
+      }
+      add_snippet('20141218', 'mbland', 'Mike Bland', 'michael.bland@gsa.gov',
+        'unused', '- Did stuff', '')
+      add_snippet('20141225', 'mbland', 'Mike Bland', 'michael.bland@gsa.gov',
+        '', '- Did stuff', '')
+      add_snippet('20141231', 'mbland', 'Mike Bland', 'michael.bland@gsa.gov',
+        'Public', '- Did stuff', '')
+      Joiner.join_data @site
       assert_equal @expected, @site.data['snippets']
```

Comment:
is it the case that `@expected` is equal to `{}` here and that is what we expect for the value of `@site.data['snippets']`? 

I don't see where the value of `@expected` is set except for at the top of the file where it is set to me an empty hash...and I would think `@site.data['snippets']` should include the snippets created in this test setup. Am I missing something?